### ISSUE TITLE
Can't load checkpoint files with tensorflow version 1.0.0. model.load()

### ISSUE THREAD

**Issue by emurina** [tflearn/tflearn#636]
model.load(model_file='/home/murl/faces_expressions/Checkpoints/-191.data-00000-of-00001') 
does not work with the tf version 1.0.0.

I get the error: Data loss: not an sstable (bad magic number): perhaps your file is in a different file format and you need to use a different restore operator?

Anyone got a solution for that?

**Comment by aymericdamien** [comment]
You have to use the name '/home/murl/faces_expressions/Checkpoints/-191' (without file extension)

**Comment by emurina** [comment]
thanks! 
it works without the extension :-)

**Comment by chandrasg** [comment]
Hi @emurina Could you please post your whole code?

I have having the following issue: 
InternalError (see above for traceback): Dst tensor is not initialized.
         [[Node: Adam/zeros_53 = Const[dtype=DT_FLOAT, value=Tensor<type: float shape: [25088,4096] values: [0 0 0]...>, _device="/job:localhost/replica:0/task:0/gpu:0"]()]]

This happens when I try to initialise: 
preds = tflearn.fully_connected(x, num_class, activation='softmax', scope='fc8')
regression = tflearn.regression(preds, optimizer='adam',
                                loss='categorical_crossentropy',
                                learning_rate=0.001)
model =tflearn.DNN(regression)
